build.gradle

```diff
@@ -233,27 +205,7 @@ dependencies {
     implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:5.2.0'
     implementation 'net.javacrumbs.shedlock:shedlock-spring:5.2.0'
 
-    // This allows you to build with a local Stairway JAR file, instead of the one from the repository
-    // For example, ./gradlew -Pstairwayjar=/Users/dd/code/stairway/build/libs/stairway-0.0.4-SNAPSHOT.jar
-    // Or define the environment variable ORG_GRADLE_PROJECT_stairwayjar
```

Comment:
These approaches to local development are out of date.  The standard is now to publish the library you're pulling in locally: https://github.com/DataBiosphere/terra-common-lib?tab=readme-ov-file#local-testing

I expanded our repositories definition above to support this.